### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by another-u** [PJLallen/OSFormer#3]
Can you provide the divided training set and test set used in your paper?

**Comment by Patrickctyyx** [comment]
We used the COD10K train set for training, COD10K test set and NC4K for testing.
The COD10K dataset is already divided in https://github.com/DengPingFan/SINet#4-proposed-cod10k-datasets

**Comment by Patrickctyyx** [comment]
We used the COD10K train set for training, COD10K test set and NC4K for testing.
The COD10K dataset is already divided in https://github.com/DengPingFan/SINet#4-proposed-cod10k-datasets

**Comment by another-u** [comment]
What you mentioned in your paper is to use 3040 pictures for training, but the cod10k train set I downloaded from the GitHub link has 6000 pictures. I don't know what's going on.

**Comment by Patrickctyyx** [comment]
Thanks for your response, following previous COD works, we only use camouflaged images for training(3040/6000) and testing(2026/4000).
We have uploded the 3040 training and 2026 testing COD10K dataset [here](https://pan.quark.cn/s/07ba3258b777). And we will upload to baidu and goole later. If you have other questions, you can contact peijl@hust.edu.cn

**Comment by PJLallen** [comment]
> What you mentioned in your paper is to use 3040 pictures for training, but the cod10k train set I downloaded from the GitHub link has 6000 pictures. I don't know what's going on.

For COD10K, we use the camouflaged images with instance-level labels for training (3,040) and tsting(2,026). We have upload instance-level COD10K in ReadMe. You can achieve it from "Datasets".
